common/source/docs/common-apm-navigation-extended-kalman-filter-overview.rst

```diff
@@ -61,6 +61,11 @@ inertial nav (Copter-3.2.1) or ahrs dead reckoning (Plane) for position control.
 
    Using the parameters above it is possible to run up to 5 AHRSs in parallel at the same time (DCMx1, EKF2x2, EKF3x2) but this can result in performance problems so if running EKF2 and EKF3 in parallel, set the IMU_MASK to reduce the total number of cores.
 
+Affinity and Lane Switching
+----------------------------
+
+EKF3 provides the feature of sensor affinity which allows the EKF cores to also use non-primary instances of sensors, specifically, Airspeed - Barometer - Compass (Magnetometer) - GPS. This allows the vehicle to better manage good quality sensors and be able to switch lanes accordingly to use the best-performing one for state estimation. For more details and configuration, refer :ref:`EKF3 Affinity and Lane Switching <dev:ek3-affinity-lane-switching>`.
+
```

Comment:
I suggest you delete this page...I have added the text to the upcoming features section since this is not in stable..

inter site references to new pages dont work locally....the inventory of cross site references is loaded from the main server and since the new page isnt there yet, it will fail to link...

Reply:
should i also remove the places I have linked this page ?

Reply:
no, just dont change this common page..I put that text already in the upcoming features stuff..
..the link in dev toc and the new page  itself can go in....

Reply:
ok, I've made the changes you suggested above to the new page. Also @tridge had asked for this page to redirect to the affinity page on Discord.

Reply:
It does, via the upcoming features section...